src/handlers/handleDownloadScript.ts

```diff
@@ -6,30 +6,39 @@ import {
   getAgentScriptEndpoint,
 } from '../utils'
 
-function createResponseWithMaxAge(oldResponse: Response, maxMaxAge: number): Response {
+function copySearchParams(oldURL: URL, newURL: URL): void {
```

Comment:
You have the same function in `src/handlers/handleIngressAPI.ts`, is it as planned?

Reply:
Yes, it was on purpose. `copySearchParams` is a one-line function. The only purpose it serves is to make the code more readable. I thought of putting this function to a common place, but I decided it is not worth it.